app/src/main/res/raw/data.json

```diff
@@ -724,6 +732,413 @@
+            "topics": ["фрукты", "еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "marmalade",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "джем",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "mayonnaise",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "майонез",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "meatball",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "фрикаделька",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "mince",
+            "topics": ["cooking", "food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "фарш",
+            "topics": ["приготовление еды", "еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "mousse",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "мусс",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "popcorn",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "попкорн",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "porridge",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "каша",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "raisin",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "изюм",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "soya",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "соя",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "syrup",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "сироп",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "toffee",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "ириска",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "vanilla",
+            "topics": ["food"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "ваниль",
+            "topics": ["еда"]
+        }}
+    ],
+    [
+        {"EN":{
```

Comment:
Duplication, please remove

Reply:
please add comment to the line where the word is located, otherwise it is not displayed in comment

Reply:
done